src/runtime_src/core/tools/common/ReportAie.cpp

```diff
@@ -200,77 +200,77 @@ populate_aie_core(boost::property_tree::ptree _pt,boost::property_tree::ptree& t
     tile.put("row", pt.get<uint32_t>("row"));
 
     std::string status;
-    for (auto& node: pt.get_child("core.Status", empty_pt)) {
+    for (auto& node: pt.get_child("core.status", empty_pt)) {
       status +=(status.empty()?"":", ")+node.second.data();
     }
     if(!status.empty())
       tile.put("core.status", status);
 
-    for (auto& node: pt.get_child("core.PC", empty_pt))
+    for (auto& node: pt.get_child("core.pc", empty_pt))
       tile.put("core.program_counter", node.second.data());
 
-    for (auto& node: pt.get_child("core.LR", empty_pt))
+    for (auto& node: pt.get_child("core.lr", empty_pt))
       tile.put("core.link_register", node.second.data());
 
-    for (auto& node: pt.get_child("core.SP", empty_pt))
+    for (auto& node: pt.get_child("core.sp", empty_pt))
       tile.put("core.stack_pointer", node.second.data());
-
+/*

```

Comment:
If this code is not needed it should be removed and NOT commented out.

Reply:
according to aie driver spec these nodes were that but in actual implementation they remove but these changes will come in future. so commented this part of code. 
But now i am removeing.